RELEASE.md

```diff
@@ -165,34 +165,34 @@ up with a release for `bpfman-operator`.
 - Create a new branch in your `bpfman` fork, for example
   `<githubuser>/release-x.y.z`, and use the new branch in the upcoming steps.
 - Make the following changes
-    - Add a new changelog for the release using the process described in
-      [Generating Release Notes](#generating-release-notes).
-    - Update the
-      [Cargo.toml](https://github.com/bpfman/bpfman/blob/main/Cargo.toml)
-      version for the workspace:
-        - `version = "x.y.z"`
-        - `bpfman = { version = "x.y.z", path = "./bpfman" }"`
-        - `bpfman-api = { version = "x.y.z", path = "./bpfman-api" }`
-        - Note: `bpfman-csi` does not need to be updated.
-    - Run `cargo generate-lockfile`
-    - Update the bpfman version in the [bpfman/examples/Makefile][examples-makefile]:
-      - VERSION ?= x.y.z
-    - Add a new `bpfman/examples/config/v0.x.y/` and
-      `bpfman/examples/config/v0.x.y-selinux/` config directory for the release
-      version by copying the latest release directory and running a search for
-      the current release and replace with the new release.
-    - Add new example config directories for any new examples added since the
-      last release.
-    - Update dispatcher tags.
-      - Modify the tag for `XDP_DISPATCHER_IMAGE` and `TC_DISPATCHER_IMAGE` in
-        `bpfman/src/lib.rs` from `latest` to the new release tag.
-      - Manually add the new release tag to the latest version of the following
-        dispatcher images:
-        - https://quay.io/repository/bpfman/xdp-dispatcher
-        - https://quay.io/repository/bpfman/tc-dispatcher
-    - Search the code and docs for the current version number without the "v"
-      (e.g., 0.5.1) and replace it with the new version number where it makes
-      sense.  (Be careful, though, because not all should be replaced.)
+  - Add a new changelog for the release using the process described in
```

Comment:
Line 154: We should add some notes about the Change Log:
* Unlike most markdown, the generated output in the github releases renders each newline in the CHANGELOG-vX.Y.Z.md file. So each paragraph needs to be on a line, or it will be rendered as seen in CHANGELOG-vX.Y.Z.md.
* The order of sections are (empty sections should be omitted):
    * Vx.y.z (with the summary)
    * What's Changed
        *  Full Changelog
    * Known Issues (usually added after the release when something is encountered and needs to be documented)
    * New Contributors
    * Contributors
    * Assets
* After the release has been generated, updates to CHANGELOG-vX.Y.Z.md file are not reflected in the Releases page in github, so the Releases page in github must be manual edited using the github GUI.

@anfredette The last bullet, "After the release has been generated" wording may not be correct. I scrubbed a bunch of the release notes to add the "Known Issues", but since I haven't been through the process, not sure when the release is rendered in the github release page. So reword as needed.

Reply:
I reworded things, but not exactly as you suggested.